packages/venia-ui/lib/components/CreateAccount/createAccount.js

```diff
@@ -18,19 +18,31 @@ import {
 } from '../../util/formValidators';
 import defaultClasses from './createAccount.css';
 import { useCreateAccount } from '@magento/peregrine/lib/talons/CreateAccount/useCreateAccount';
+import CREATE_ACCOUNT_MUTATION from '../../queries/createAccount.graphql';
 
 const LEAD =
     'Check out faster, use multiple addresses, track orders and more by creating an account!';
 
 const CreateAccount = props => {
     const talonProps = useCreateAccount({
-        initialValues: props.initialValues
+        initialValues: props.initialValues,
+        query: CREATE_ACCOUNT_MUTATION,
```

Comment:
Only because in GraphQL parlance "queries" and "mutations" are two distinctly separate things, this reads funny.

Reply:
Yea -- I thought about changing the import name. I don't really think the UI needs to care about whether the query is a mutation/query/subscription.

I'd like to get @jimbo's take on naming here.

Reply:
👍 pending @jimbo 's verdict.

Reply:
> Whether or not the front end splits these things up into `queries` and `mutations` directories is a discussion worth having, but GraphQL and our Magento GraphQL schema does separate them, so I assumed we would want to too, if only to reduce confusion when going back and forth.

I'm pretty convinced by @supernova-at's argument here. As long as we have a `queries` folder, it makes sense have a `mutations` folder, since they're separate entities.

> Since this is the first mutation, I'm wondering if we shouldn't create a lib/mutations/ folder.

That said, this is a reminder that there is no good reason to have a `queries` folder in the first place. Queries (and mutations) are supposed to be highly tailored to the component using them; it's not considered an optimization to reuse a query, so there's no value in keeping them in a shared folder.

Ideally, queries and mutations would reside in the component folder alongside `.js` and `.css` component files. If we're not going to do that, a `mutations` directory is fine.

Reply:
> That said, this is a reminder that there is no good reason to have a queries folder in the first place.

I agree. I think we moved them into the `queries/` folder maybe to make it easier on some tooling? I'm not sure.

How about this compromise - we merge this as is and I open an issue to move all the queries/mutations to the component folders where they are used.

Reply:
Made https://github.com/magento/pwa-studio/issues/1928 :D